index.js

```diff
@@ -0,0 +1,68 @@
+var play = prompt("What's your move?  --  (R)ock, (P)aper or (S)cissors");
+var x;
+
+if (play === "r") {
+  x = "rock";
+  console.log("Player " + x);
+}
+else if (play === "p") {
+  x = "paper";
+  console.log("player " + x);
+}
+else if (play === "s") {
+  x = "scissors";
+  console.log("player " + x);
+}
+
+
+var cpuChance = Math.random();
+var y;
+
+if (cpuChance < 0.34) {
+  y = "rock";
+  console.log("cpu rock");
+}
+else if (cpuChance <= 0.67) {
+  y = "paper";
+  console.log("cpu paper");
+}
+else {
+  y = "scissors";
+  console.log("cpu scissors");
+}
+
+
+
+
+var match = function (x, y) {
+  if (x === y) {
+    return "It's a tie...";
```

Comment:
This function returns a string saying who won, lost or tied correctly, but this information is never actually outputted to the player.